Tests/SwiftProtobufTests/Test_Struct.swift

```diff
@@ -171,18 +161,21 @@ class Test_JSON_ListValue: XCTestCase, PBTestHelpers {
 class Test_JSON_Value: XCTestCase, PBTestHelpers {
     typealias MessageTestType = Google_Protobuf_Value
 
-    func testValue_default() throws {
+    func testValue_emptyShouldThrow() throws {
         let empty = Google_Protobuf_Value()
-        let emptyJSON = try empty.jsonString()
-        XCTAssertEqual("null", emptyJSON)
-
-        XCTAssertEqual(empty, Google_Protobuf_Value(nullValue: ()))
-        XCTAssertNotEqual(empty, Google_Protobuf_Value(numberValue: 1))
+        do {
+            _ = try empty.jsonString()
+            XCTFail("Encoding should have thrown .missingValue, but it succeeded")
+        } catch let e as JSONEncodingError {
+            XCTAssertEqual(e, JSONEncodingError.missingValue)
+        } catch {
+            XCTFail("Encoding should have thrown .missingValue, but instead it threw: \(error)")
+        }
     }
 
```

Comment:
Should we test other behaviors of a default `Google_Protobuf_Value()` object?

For example, I presume it should successfully binary encode (to a zero-length value), should successfully text encode (to an empty "{}"), and should compare not-equal to `nil`.